js/app.js

```diff
@@ -2,123 +2,212 @@
 
 //Time Stamp Array for all stores
 var timeSale = ['6am ', '7am ', '8am ' , '9am ', '10am ', '11am ', '12pm ', '1pm ', '2pm ','3pm ', '4pm ', '5pm ','6pm ', '7pm ', '8pm '];
-
-
-//Object 1st and Pike
-var firstPike = {
-  minCust: 23,
-  maxCust: 65,
-  avgCust: 6.3,
-  cookieAmount: function(){
-    var ulElGet = document.getElementById('Pike');
-    var cookieTotal = 0;
-    for(var i = 0; i < timeSale.length; i++){
-      var cookieHour = Math.ceil(this.avgCust * (Math.random() * (this.maxCust - this.minCust) + this.minCust));
-      cookieTotal += cookieHour;
-      var liEl = document.createElement('li');
-      liEl.textContent = 'Cookies sold at  ' + timeSale[i] +  ' : ' + cookieHour;
-      ulElGet.appendChild(liEl);
-    } 
-    var liEl1 = document.createElement('li');
-    liEl1.textContent = 'Total number of cookie sold today  ' + cookieTotal;
-    ulElGet.appendChild(liEl1);
+var allStore = [];
+var storeTable = document.getElementById('salmon_table');
+
+function StoreLoc(sTname, minCust, maxCust, avgCust){
+  this.sTname = sTname;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgCust = avgCust;
+  allStore.push(this);
+}
+
+StoreLoc.prototype.cookieAmount = function(){
+  var cookieTotal = 0;
+  var tbTr = document.createElement('tr');
+  var tbTd = document.createElement('td');
+  tbTd.textContent = this.sTname;
+  tbTr.appendChild(tbTd);
+  for(var i = 0; i < timeSale.length; i++){
+    var cookieHour = Math.ceil(this.avgCust * (Math.random() * (this.maxCust - this.minCust) + this.minCust));
+    cookieTotal += cookieHour;
+    tbTd = document.createElement('td');
+    tbTd.textContent = cookieHour;
+    tbTr.appendChild(tbTd);
   }
+  tbTd = document.createElement('td');
+  tbTd.textContent = cookieTotal;
+  tbTr.appendChild(tbTd);
+  storeTable.appendChild(tbTr);
+  return cookieTotal;
```

Comment:
You have a `return` statement here, but you never use this value. You should only be returning a value if you intend to use it at some point.